src/rospack.cpp

```diff
@@ -2231,16 +2223,6 @@ Rospack::Rospack() :
 {
 }
 
-Rosstackage::~Rosstackage()
-{
-  for(std::tr1::unordered_map<std::string, Stackage*>::const_iterator it = stackages_.begin();
-      it != stackages_.end();
-      ++it)
-  {
-    delete it->second;
-  }
-}
-
```

Comment:
Please don't move the location of the destructor around. It is the same order as in the header and results in unnecessary changes.